osidb/models.py

```diff
@@ -2109,7 +2109,7 @@ def create_cvss(affect, issuer, version, **extra_fields):
             )
 
 
-class CVSS(ACLMixin, BugzillaSyncMixin, TrackingMixin):
+class CVSS(ACLMixin, BugzillaSyncMixin, TrackingMixin, NullStrFieldsMixin):
```

Comment:
nitpick: When there is no specific reason for a certain order, we should stick with the alphabet

Reply:
Thanks, fixed.